release-team/release-team-selection.md

```diff
@@ -0,0 +1,99 @@
+
+While there isn't a strict restriction on the amount of shadows, we've found three shadows per role to be a reasonable upper bound. However, as shadowing is intended to be a mentor / mentee journey, it is important that role leads only accept an amount of shadows that they feel that can reasonably mentor within the scope of a release cycle.
+
+If there are more contributors interested in shadowing once hitting that upper bound, we welcome them to sit in on Release Team meetings in preparation for shadowing in a future release cycle.
+
+In Kubernetes 1.14, we begun handling shadow selection using a questionnaire, as opposed to the first-come, first-served GitHub issue approach that we had adopted during previous release cycles ([1.12](https://github.com/kubernetes/sig-release/issues/167), [1.13](https://github.com/kubernetes/sig-release/issues/280).
+
+Moving to a [questionnaire](#sample-questionnaire) / survey format will allow us to capture more of the information required to vet potential shadows and well as begin to generate metrics around some of the Release Team processes.
+
+Questionnaire creation will be facilitated by SIG Release Chairs or a delegated coordinator to ensure consistency across release cycles.
+
+Following submissions, the questionnaires will be reviewed by role leads of the incoming Release Team.
+
+Volunteers that meet the requirements of the respective role will be contacted by those leads via some convenient mechanism (Slack, video chat, etc.) to further discuss.
+
+After vetting the volunteers for their roles, role leads should make a final decision on selected shadows with the incoming Release Team Lead.
+
+
+### Sample Questionnaire
+
+Thank you for volunteering for the Kubernetes 1.14 Release Team! 
+
+This questionnaire is meant to learn a little bit more about you, your journey with Kubernetes, your understanding of the Release Team processes and procedures, as well as to determine if you're a good fit for the current Release Team. Please note that you are expected to be inexperienced in some areas, as a role shadow is a mentorship opportunity.
+
+#### Logistics
+
+- What [Release Team role(s)](https://git.k8s.io/sig-release/release-team#kubernetes-release-team-roles) are you interested in shadowing? _[Select up to two (2)]_
+- The [Role Handbooks](https://git.k8s.io/sig-release/release-team/role-handbooks) defines the tasks and responsibilities of each role for this release cycle. Are you prepared to assist in fulfilling these duties? Please read the handbook relevant to the role you're interested in before answering.
+- Shadowing Release Team roles requires a substantial time commitment, especially during the periods specified in your role handbook. The time requirements specified are estimates and depending on the need, you may need to dedicate more time than initially estimated. This may include attending video meetings at inconvenient times. Is this time you can commit to?
```

Comment:
I would prefer more precise wording than "may require... inconvenient times". It's one thing if we're talking once a week call at 8pm, another if it's a daily burndown meeting at 1am (which can happen if someone is based in Asia). Expecting all candidates to be fully available at undisclosed times and frequency feels unnecessary, and may end up rather exclusionary, given that the meetings are always scheduled in PST-friendly time.

Reply:
@aleksandra-malinowska -- Good points. Where do you think we can strike a balance, while being careful not to be exclusionary?

Some things I think/know/feel:
- We have some flexibility on who attends the meetings (lead, shadow, some combination of both). Doesn't need to be everyone, all the time. As long as the role has coverage for required activities, then I think that's fine.
- PST-friendly is unfortunately the nature of the beast at the moment, though we should see what we can do to make that more inclusive for everyone. Do you think it's enough to mention `US Pacific` in the survey?
- We used `may require... inconvenient times` to suggest that the times could also be convenient for you. Consider someone who could only volunteer during working hours, due to other commitments. If their working hours happen to be US-centric, then the RT meetings actually could be convenient for them. I want to make sure the language is inclusive of people with commitment restrictions, not just timezone restrictions.
- It's hard to nail down exactly when meetings will be. We have, in the past, taken polls (h/t @AishSundar / @tpepper) to determine when we should hold the meetings, but that's really only possible after the team is formed and the RT Lead has a chance to chat with people about timezones. That said, I wouldn't feel comfortable implying times ahead of time.

Reply:
We do need to be thinking about formalizing release team as more than PST-dominated.  By leaving it open it's almost self-fulfilling in who volunteers and follows through.

Reply:
Agreed.